From acdfd2b06526b056195f438680e8f1d6b3a5a0b8 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 28 Mar 2023 15:34:45 +0200
Subject: [PATCH 0156/1141] arm64: defconfig: Enable the STM32MP25 ComboPHY

STM32 ComboPHY to generate the PCIe or USB3 refClk.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I63e907dd9fc9d248b6cd1908a95c6e3069686ec5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/306765
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 0b4502d70afd..f690097f9ce8 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -13,6 +13,7 @@ CONFIG_NET_CLS_U32=m
 CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_SKBPRIO=m
+CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
-- 
2.39.2

